lib/linting/schemas/core.js

```diff
@@ -17,7 +17,7 @@
 const tags = require('./definitions/tags');
 
 module.exports = {
-  $id: 'https://example.com/person.schema.json',
+  $id: 'https://chromeos.dev/core.schema.json',
```

Comment:
This isn't strictly needed, but by spec it doesn't need to resolve so I changed it to our site.